dra-evolution/pkg/api/capacity_types.go

```diff
@@ -68,6 +68,11 @@ type ResourcePoolSpec struct {
 	// +optional
 	SharedCapacity []SharedCapacity `json:"sharedCapacity,omitempty"`
 
+	// Attributes contains common device attributes that are the same
+	// for all devices in the pool, unless a device specifically over
```

Comment:
"the pool" or "the slice"?

The prototype is not up-to-date here :cry: We should update it to match the current KEP before doing further prototyping.

Reply:
Yeah. I didn't want to do that yet, I figured it could be done and I could rebase...